docs/user_guide/metrics.md

```diff
@@ -133,36 +133,40 @@ If building Triton locally, the `TRITON_ENABLE_METRICS_CPU` CMake build flag can
 
 ## Response Cache Metrics
 
-Compute latency metrics in the [Inference Request Metrics table](#inference-request-metrics) above are calculated for the
-time spent in model inference backends. If the response cache is enabled for a
-given model (see [Response Cache](response_cache.md)
+Cache metrics can be reported in two ways:
+
+1. A base set of cache metrics will be reported
+by Triton directly, such as the cache hit/miss counts and durations described
+below.
+
+2. As of 23.02, additional cache metrics may be reported depending on the 
+[cache implementation](response_cache.md#cache-implementations) 
+being used through Triton's [Metrics API](#custom-metrics).
+
+### Triton-reported Response Cache Metrics
+
+Compute latency metrics in the 
+[Inference Request Metrics table](#inference-request-metrics) above are 
+calculated for the time spent in model inference backends. If the response 
+cache is enabled for a given model (see [Response Cache](response_cache.md)
 docs for more info), total inference times may be affected by response cache
 lookup times.
 
-On cache hits, "Cache Hit Lookup Time" indicates the time spent looking up the
+On cache hits, "Cache Hit Time" indicates the time spent looking up the
 response, and "Compute Input Time" /  "Compute Time" / "Compute Output Time"
 are not recorded.
 
-On cache misses, "Cache Miss Lookup Time" indicates the time spent looking up
-the request hash and "Cache Miss Insertion Time" indicates the time spent
-inserting the computed output tensor data into the cache. Otherwise, "Compute
-Input Time" /  "Compute Time" / "Compute Output Time" will be recorded as usual.
+On cache misses, "Cache Miss Time" indicates the time spent looking up
+the request hash and inserting the computed output tensor data into the cache. 
+Otherwise, "Compute Input Time" /  "Compute Time" / "Compute Output Time" will
+be recorded as usual.
 
 |Category      |Metric          |Metric Name |Description                            |Granularity|Frequency    |
```

Comment:
It looks like you no longer use the category column, so you may want to get rid of it.

Reply:
Good catch, fixed the column